index.js

```diff
@@ -47,4 +47,14 @@ function createError (code, message, statusCode = 500, Base = Error) {
   return FastifyError
 }
 
+function isFastifyError (obj) {
+  return (
+    Object.prototype.toString.call(obj) === '[object Error]' &&
+    obj.name === 'FastifyError'
```

Comment:
Please review the issue again. I requested using [toStringTag](http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) to facilitate the check.